source/doi_howto.rst

```diff
@@ -0,0 +1,124 @@
+                            data                extent it overlaps with discovery metadata, but more so it really extends discovery
+                                                metadata. Site metadata can be used for observation network design.
+   ======================== =================== ======================================================================================== =================== 
+
+Delivering ACDD and CF compliant data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As mentioned above, discovery metadata are those information which are used to find what a user is looking for. Following the Attribute Convention for Data Discovery (`ACDD <http://wiki.esipfed.org/index.php/Attribute_Convention_for_Data_Discovery_1-3>`_), there are no mandatory fields, although some are highly recommended, and as general rule, the more information is given, the easier 
+for a potential data user to find relevant data. It is recommended to use ACDD in version 1.3 or higher.  
+
+Once the data are found, it is essential to be able to understand  unequivocally its content. The `CF conventions <http://cfconventions.org/>`_ define metadata that provide a definitive description of what the data in each variable represents, and the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities. It is recommended to use CF in version 1.6 or higher.
+
+When correct, all information can be valuable. 
+
+Compliance with the above mentioned standard can be checked using online validation tools. A locally implemented version of IOOS compliance checker can be found on the adc.met.no portal (https://adc.met.no/dataset_validation/form), while other online tools are also listed below:
+
+* NetCDF/CF and ACDD checkers:
+
+ - http://cfconventions.org/compliance-checker.html
+ - http://puma.nerc.ac.uk/cgi-bin/cf-checker.pl
+ - Source code (standalone Python) https://pypi.python.org/pypi/cfchecker
+ - https://podaac-uat.jpl.nasa.gov/mcc/ (Both CF and ACDD)
+ - https://data.ioos.us/compliance/index.html
+ - https://github.com/ioos/compliance-checker(Both CF and ACDD)
+
+Make a Dataset available
+------------------------
+Once data are prepared, with necessary metadata as described above, these data should be made available, as for example through THREDDS or HYRAX data servers. 
+Data should be structured to allow for aggregation if needed, to reflect a useful granularity, and selected services, depending on the data provided, should be 
+exposed on the data server, as for example OPeNDAP access to data to allow remote data stream. 
+
+.. note:: More on how to configure a dataset relatates to the issue: `Document how to make data available on thredds (or other DAP server) <https://app.zenhub.com/workspaces/s-enda-5e7b1b4acfc5ca185fb6b53c/issues/metno/s-enda-documentation/139>`_
```

Comment:
It is better to link to the issue on github - this one is probably not visible unless people are logged in to zenhub

Reply:
Sure, I'll fix that. I'll wait for Martin as well, then I can just merge.